### ISSUE TITLE
Use User Official Email To Send Email Notification To Customer

### ISSUE THREAD

**Issue by olawalejuwonm** [hngi/sentry-my-customer-api#79]
**What Functionality / Service / Endpoint would you like to add?**
This will add the functionality of user to send email with their personal/official email they used to register

**Describe this Functionality / Service / Endpoint**
A clear and concise description of what you want to add.
In the transporter used for sending the email, the email present is a static email(not depending on the user) and simply a temporary one, implementing this will cause the endpoint to authenticate user, get user details from the request header and then use the email to send email notification to users

**Duplicate prevention**
Have you checked that this issue has not been opened by anyone  else?
Yes

**Describe how to intend to go about adding this feature**
Describe the approach you will take in adding this feature. 

- [x] Get user to login

- [x] Authenticate the user

- [x] Assign token

- [x] Get details from request.user

- [x] Extract the email from request header

- [x] Add email to the transporter

**Slack username**
@OlawaleJuwonM

**Additional context**
`const transporter = nodemailer.createTransport({
            service: 'gmail',
            auth: {
                user: 'melijah200@gmail.com',
                pass: 'emailpasswordgoeshere'
            }
        });`

**Comment by Daemonicc** [comment]
will you trust a site  enough to give out your email password?

**Comment by chinazaaa** [comment]
Are you fixing the issue?

**Comment by chinazaaa** [comment]
Approved by @chinazaaa

**Comment by naijauser** [comment]
Approved by @naijauser

**Comment by chinazaaa** [comment]
approved by @chinazaaa

**Comment by naijauser** [comment]
The only thing that needs to be done is to get the `to`, `subject `and `text `properties in the `params `object

```
const params = {
                        from: 'melijah200@gmail.com',
                        to: 'melijah200@yahoo.com',
                        subject: 'Sending Email using Node.js',
                        text: 'That was easy!'
 };
```

to use the `recipient`, `subject ` and `text ` variables.

```
const recipient = foundCustomer.email,
                        subject = req.body.subject,
                        text = req.body.text;
```
We'll go ahead and approve this. Please note that this is the only change expected.

**Comment by naijauser** [comment]
Approved by @naijauser

**Comment by chinazaaa** [comment]
approved by @chinazaaa

**Comment by iAmServer** [comment]
Can the microapi mailer be used for this feature?

**Comment by olawalejuwonm** [comment]
am solving this issue @naijauser

**Comment by olawalejuwonm** [comment]
this has already been added @iAmServer  just some additional features needed

**Comment by olawalejuwonm** [comment]
@naijauser @chinazaaa  I have completed this but has not been merged since
https://github.com/hngi/sentry-my-customer-api/pull/120

**Comment by olawalejuwonm** [comment]
@chinazaaa

**Comment by emasontech** [comment]
I have fixed this issue #79, no need for token. Issue was the structure in dB solution was to convert the user object coming from dB to  json then access the email..
